package/kernel/mac80211/patches/020-21-rt2800-fix-LNA-gain-assignment-for-MT7620.patch

```diff
@@ -38,10 +38,7 @@ Signed-off-by: Daniel Golle <daniel@makrotopia.org>
  
  		/* AGC init */
 -		reg = (rf->channel <= 14 ? 0x1c : 0x24) + 2 * rt2x00dev->lna_gain;
-+		if (rt2x00_rt(rt2x00dev, RT6352))
-+			reg = 0x04;
-+		else
-+			reg = rf->channel <= 14 ? 0x1c : 0x24;
```

Comment:
See `RT6352_RTMPSetAGCInitValue` in the vendor driver.
> ```
> R66 = 0x04 + 2 * GET_LNA_GAIN(pAd);
> ```

represented in the case for RT6352 in `rt2800_config_channel` in rt2x00:
> ```
>                if (rt2x00_rt(rt2x00dev, RT6352))
>                        reg = 0x04;
>                else
>                        reg = rf->channel <= 14 ? 0x1c : 0x24;
> 
>                reg += 2 * rt2x00dev->lna_gain;
> ```

This PR basically suggests to revert the BBP R66 related part of `020-21-rt2800-fix-LNA-gain-assignment-for-MT7620.patch`. (it still keeps the modification of the addition of the LNA gain syntactically, hence the PR only semantically reverts that part of the patch and leaves behind some cosmetic glitch)